applications/nrf_desktop/README.rst

```diff
@@ -331,6 +331,44 @@ Add the ``pm_static_${CMAKE_BUILD_TYPE}.yml`` partition manager configuration fi
 
 For more information about how to configure the flash memory layout using the partition manager, see :ref:`partition_manager`.
 
+Device Firmware Upgrade
+***********************
+
+The nRF Desktop application uses background image transfer for the DFU process.
+The following elements take part in the firmware update process:
+
+    * `Update image generation`_
+    * `Update image transfer`_
+    * `Update image verification and swap`_
+
```

Comment:
Please add a line in line 344 to shortly describe the result of these three steps.
```
At the end of these three stages, the nRF Desktop application will be rebooted with the new firmware package installed.
```

Reply:
Added